filebeat/docs/inputs/input-common-file-options.asciidoc

```diff
@@ -397,3 +401,34 @@ file that hasn't been harvested for a longer period of time.
 This configuration option applies per input. You can use this option to
 indirectly set higher priorities on certain inputs by assigning a higher
 limit of harvesters.
+
+[float]
+===== `file_identity`
+
+Different `file_identity` methods can be configured to suit the
+environment you are collecting log messages from.
+
+
+*`inode_deviceid`*:: The default behaviour of {beatname_uc} is to differentiate
+between files using their inodes and device ids.
+
+[source,yaml]
+----
+file_identity.inode_deviceid: ~
+----
+
+*`path`*:: To identify files based on their paths use this strategy.
```

Comment:
We should add a warning here that this strategy is only viable if new files are put into the directory, but will fail tracking renamed files (log rotation use-case).